### DISCUSSION TITLE
其实macos不需要docker也能用

### DISCUSSION THREAD

**Discussion by mmfmkuang** [mikumifa/biliTickerBuy#387] [聊天专区]
其实macos不需要docker也能用，依赖的本地过验证码pip如果装不上的话只要在https://rustup.rs 安装好rust后就能装上了，然后后面一切OK

**Comment by dillzhu** [comment]
确实，Linux也可以，装好rust且配置好解释器和环境就好

**Comment by mikumifa** [comment]
用docker比较无脑，适合新人
